data/json/npcs/isherwood_farm/NPC_Eddie_Isherwood.json

```
@@ -261,12 +261,12 @@
   {
     "id": "MISSION_ISHERWOOD_EDDIE_3",
     "type": "mission_definition",
-    "name": { "str": "Gather 1000 units of clay" },
+    "name": { "str": "Gather 50 units of clay" },
```

```suggestion
    "name": { "str": "Gather 100 units of clay" },
```
Applying changes PatrickLundell suggested.